gadgets/trace_tcpconnect/README.mdx

```diff
@@ -28,10 +30,120 @@ Running the gadget:
     </TabItem>
 </Tabs>
 
-## Flags
+## Guide
 
-No flags.
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
```

Comment:
Improvement for a follow-up PR:

Use here the example we had in the built-in version here: https://www.inspektor-gadget.io/docs/latest/gadgets/builtin/trace/tcpconnect#on-kubernetes (After updating it with new error field 🙂)